data/json/monsters/mammal.json

```diff
@@ -1759,6 +1759,83 @@
     },
     "flags": [ "ANIMAL", "DOGFOOD", "CANPLAY", "HEARS", "KEENNOSE", "PATH_AVOID_DANGER_1", "SEES", "SMELLS", "WARM" ]
   },
+  {
+    "id": "mon_dog_gswiss",
+    "copy-from": "mon_dog",
+    "type": "MONSTER",
+    "name": { "str_sp": "Greater Swiss" },
+    "description": "The biggest of four breeds of Swiss mountain dogs.  The Greater Swiss mountain dog was originally meant to pull carts, herd cattle and guard the homestead.  This one seems feral, but perhaps you could try taming it?",
+    "volume": "55000 ml",
+    "weight": "55 kg",
+    "hp": 48,
+    "speed": 130,
+    "aggression": 4,
+    "morale": 8,
+    "melee_skill": 3,
+    "melee_dice_sides": 2,
+    "melee_damage": [ { "damage_type": "cut", "amount": 6 } ],
```

Comment:
6 cut is too much for doggo claws, most of their damage output should come from their bite.